core/src/scss/utilities/functions/z-index/_z-index.scss

```diff
@@ -0,0 +1,18 @@
+@charset "UTF-8";
+
+// z($level)
+//
+// A function to help us get the z-index from the level from the map
+//
+// $level - the name of the z-index level
+//
+// Style guide: Functions.z-index.level
+//
+
+
+@function z($level) {
+  @if map-has-key($z-levels, $level) {
```

Comment:
Perhaps make the `$level` param default to `default` as that is an option in the map.